src/main/java/mat/client/cqlworkspace/AddFunctionArgumentDialogBox.java

```diff
@@ -85,7 +89,11 @@ public static  void showArgumentDialogBox(final CQLFunctionArgument functionArg,
 		listAllDataTypes.setWidth("290px");
 		listAllDataTypes.addItem(MatContext.PLEASE_SELECT);
 		for (int i = 0; i < allCqlDataType.size(); i++) {
-			listAllDataTypes.addItem(allCqlDataType.get(i));
+			if (MeasureDetailsUtil.FHIR.equalsIgnoreCase(currentModelType) && CQLWorkSpaceConstants.CQL_MODEL_DATA_TYPE.equalsIgnoreCase(allCqlDataType.get(i))) {
```

Comment:
Could you please explain that part of the logic?
Is the purpose of that just to correct upper/lower case of a current data type item?

Reply:
ok, got it. ignore my previous comment